tests/unit/test_runner_metrics.py

```diff
@@ -18,6 +22,10 @@
     RunnerMetrics,
 )
 
+TEST_JOB_CREATED_AT = "2021-10-01T00:00:00Z"
+TEST_JOB_STARTED_AT = "2021-10-01T01:00:00Z"
+TEST_QUEUE_DURATION = 3600
```

Comment:
Do you think these would be better as fixtures? That way we have a consistent way of supplying input to our tests